operator/src/test/java/oracle/kubernetes/operator/WatcherTestBase.java

```diff
@@ -152,6 +160,36 @@ public void afterHttpGoneError_nextRequestSendsIncludedResourceVersion() {
     }
   }
 
+  @SuppressWarnings({"unchecked", "rawtypes"})
```

Comment:
Usually, it is better to let any thrown exceptions propagate from the method, so that they indicate any problems. Catching them can just hide those problems. So remove the catch class, and just declare the needed exceptions on the methods' throw clauses.

Reply:
Beyond that, everything looks good.

Reply:
Funny...  I just copied these tests, but I'll fix.